README.md

```diff
@@ -84,6 +87,7 @@ Options:
               "builder": "angular-cli-builders:custom-webpack-server"
               "options": {
                     "webpackConfigPath": "./extra-webpack.config.js",
+                    "mergeStrategy": { "plugins": "prepend" },
```

Comment:
Same as above but can be prepend (to describe different behavior). It would be also nice to give here another example rather than plugins (for example loaders).